spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/VaultEnvironmentProperties.java

```diff
@@ -73,6 +73,11 @@
 	 */
 	private int kvVersion = 1;
 
+	/**
+	 * Value namespace. default to root
+	 */
+	private String namespace = "";
```

Comment:
the default looks to be the empty string

Reply:
Yeah, I think it should be null since this is optional

Reply:
OK the comment confused me.  It is not that default value should be `root` its that when no namespace is specified we use root.  Some clarification in the comment would be good.

Reply:
I adjusted the comment and i namespace tests will fail because opensource vault binary doesn't have namespace feature. It's only available in the enterprise license edition